lib/matplotlib/tests/test_font_manager.py

```diff
@@ -310,5 +310,7 @@ def test_get_font_names():
             pass
     available_fonts = sorted(list(set(ttf_fonts)))
     mpl_font_names = sorted(fontManager.get_font_names())
+    assert set(available_fonts) - set(mpl_font_names) == set()
+    assert set(mpl_font_names) - set(available_fonts) == set()
```

Comment:
is there a reason not to
```suggestion
    assert set(available_fonts) == set(mpl_font_names)
    assert set(mpl_font_names) == set(available_fonts)
```

Reply:
Will that show the difference between the sets? (Better asking than doing another build just to, maybe, see the effect.)

Reply:
I figure that out... Seems like one of them is enough then.